### ISSUE TITLE
Source for Simple CGM Time?

### ISSUE THREAD

**Issue by toxuin** [tynbendad/simplecgmanalog#7]
Hello,

Do you have sources for the "Simple CGM Time" watchface? JS source code looks to be similar between it and this watchface, but not the same.

I know it's an old project, but I enjoy that watchface daily and it has become my favourite thing about owning a Pebble.

Even if you don't have the source, or would like to not publish it, I want to say kind thank you for your awesome watchfaces that make my - as well as so many people too I bet - life a bit more enjoyable.

**Comment by tynbendad** [comment]
yes, i believe i downloaded all my source when pebble told us they were disabling their online developer environment.  i've now uploaded it all here, https://github.com/tynbendad/pebble-archive - sorry i'm not sure what the difference was between "simple cgm time" and "cgm simple time" sources - i may have been in the middle of one or testing something, perhaps you can figure it out and update me here.

**Comment by toxuin** [comment]
Thanks a lot!

From a cursory overview, those two watchfaces have very similar code, but looks like `simple_cgm_time` is quite a bit ahead of `cgm_simple_time` :)

Again, thanks for your work!
